src/services/auth/index.tsx

```diff
@@ -172,5 +191,95 @@ export const mockValidateOTP = async (
   };
 };
 
+/** Error message returned by endpoints without a /auth/logout handler */
+const oldBackendError =
+  "Authorization header requires 'Credential' parameter. Authorization header requires 'Signature' parameter. Authorization header requires 'SignedHeaders' parameter. Authorization header requires existence of either a 'X-Amz-Date' or a 'Date' header. Authorization=";
+
+export const liveCallLogout = async (
+  sessionToken: string,
+  operatorToken: string,
+  endpoint: string
+): Promise<void> => {
+  const payload = { sessionToken, operatorToken };
+
+  try {
+    await fetchWithValidator(LogoutResponse, `${endpoint}/auth/logout`, {
+      method: "POST",
+      headers: {
+        Authorization: sessionToken,
+      },
+      body: JSON.stringify({
+        operatorToken,
+      }),
+    });
+    Sentry.addBreadcrumb({
+      category: "logout",
+      message: JSON.stringify({
+        status: "success",
+        ...payload,
+        endpoint,
+      }),
+    });
+    return;
+  } catch (e) {
+    if (e instanceof NetworkError) {
+      throw e;
+    } else if (
+      e instanceof SessionError ||
+      e.message === "Auth token is not currently valid" ||
+      e.message === oldBackendError + sessionToken
+    ) {
+      // for this case the session token is already invalidated or past the valid date
+      // logoutTodo: is it safe to keep session token there if already past validity date
```

Comment:
ok! It was more for me to differentiate TODOs that were related to this PR specifically, but I can rename it to something like `TODO: <logout>` that will still be recognised by extensions

Reply:
If a sessionToken on the database is already past the valid date, the logout handler on the backend side will fail. However, I have written the FE to just discard the sessionToken on the FE side since the sessionToken is expired on BE and unusable anyway.

Reply:
I thought the finding says to remove it from the backend?

Reply:
The finding was about session tokens that are still able to be used even though the user has logged out from the app. In this case, because the validity of the session token has expired, we will not be able to use the session token as the authenticatedHandler will block at the middleware, so an attacker also can't use the session token. The side effect is also that we also can't use the session token to call the handler to log out.